docs/faq.md

```diff
@@ -1,5 +1,12 @@
 # FAQ
 
+## Is there any specific tooling that can help me create/understand my disruptions?
+
+Yes! Take a look at [Chaosli](../cli/chaosli/README.md). This is a command line interface that has many features which include:
+explaining your disruption configuration is a human digestible way.
```

Comment:
Take a look at the preview, this doesn't line break in the way I think you expect?